0.11.0/functional.html

```diff
@@ -1495,13 +1495,6 @@ <h3><span class="hidden-section">spectrogram</span><a class="headerlink" href="#
 <dd class="field-odd"><p>Tensor</p>
 </dd>
 </dl>
-<dl>
-<dt>Tutorials using <code class="docutils literal notranslate"><span class="pre">spectrogram</span></code>:</dt><dd><div class="sphx-glr-thumbcontainer" tooltip="torchaudio implements feature extractions commonly used in the audio domain. They are available..."><div class="figure align-default" id="id13">
```

Comment:
I was wondering why this should be removed, but now looking at the tutorial, this should not be in place in the first place. 

It should be either `torchaudio.transforms.Spectrogram()` or `torchaudio.functional.spectrogram()`.

https://pytorch.org/audio/stable/tutorials/audio_feature_extractions_tutorial.html#spectrogram

https://github.com/pytorch/audio/blob/4b47412e91c22d75a1d5f4ea5f71e0ceaa26efd7/examples/tutorials/audio_feature_extractions_tutorial.py#L210-L216